stdlib/src/commonMain/kotlin/it/unibo/collektive/stdlib/spreading/Spreading.kt

```diff
@@ -44,8 +44,15 @@ inline fun <ID : Any, reified Distance : Comparable<Distance>> Aggregate<ID>.dis
 /**
  * Computes the hop distance from the closest [source].
```

Comment:
It could be useful to state in the KDoc that this function could cap to `MAX_VALUE` in the `accumulateDistance`, leading to unexpected behavior. What do you think @DanySK @angelacorte?